flask_resty/routing.py

```diff
@@ -1,21 +0,0 @@
-from werkzeug.routing import Rule
-
-try:
-    from werkzeug.routing import RequestPath
-except ImportError:  # pragma: no cover
-    # werkzeug<1.0
-    from werkzeug.routing import RequestSlash as RequestPath
-
-# -----------------------------------------------------------------------------
-
-
-class StrictRule(Rule):
```

Comment:
Decided to remove this as this doesn't work with latest and we don't use it at all on our end.

Reply:
I don't see any references here either